src/Compilation.zig

```diff
@@ -3246,13 +3253,6 @@ fn processOneJob(comp: *Compilation, job: Job) !void {
 
             const module = comp.bin_file.options.module.?;
             module.semaPkg(pkg) catch |err| switch (err) {
-                error.CurrentWorkingDirectoryUnlinked,
-                error.Unexpected,
-                => comp.lockAndSetMiscFailure(
-                    .analyze_pkg,
-                    "unexpected problem analyzing package '{s}'",
-                    .{pkg.root_src_path},
-                ),
```

Comment:
I am getting this error when compiling the PR:

```
            module.semaPkg(pkg) catch |err| switch (err) {
                                            ^~~~~~
/home/user/zig/src/Compilation.zig:3255:45: note: unhandled error value: 'error.CurrentWorkingDirectoryUnlinked'
/home/user/zig/src/Compilation.zig:3255:45: note: unhandled error value: 'error.Unexpected'
referenced by:
    performAllTheWork: /home/user/zig/src/Compilation.zig:3087:17
    update: /home/user/zig/src/Compilation.zig:2396:13
    remaining reference traces hidden; use '-freference-trace' to see all reference traces

```

I am quite unsure how the CI passes this.

Reply:
I suspect you have mixed the new binary with the old standard library. It would explain this error as well as the regression noted below regarding absolute ZIG_LIB_DIR.

Reply:
You are right, not sure how this happened. I built zig like I always do. Hmm. Indeed this was a false alarm (and the reported issue below), re-checked with master.